general:
  frequency: 40
topics:
  cmd_vel: "radiolink/cmd_vel"
  cmd_pose: "radiolink/cmd_pose"
  cmd_ef_pose: "radiolink/ef_position/command"
  cmd_joint_pos: "radiolink/joint_group_position_controller/command" 
  status: "radiolink/status"
gait:
  max_speed_x: 0.8
  max_speed_y: 0.2
  max_speed_z: 0.85
  stride_height: 0.045
pose:
  max_angle_x: 0.43
  max_angle_y: 0.25
  max_angle_z: 0.33
  max_lin_x: 0.1
  max_lin_y: 0.1
  max_lin_z: 0.05
  min_lin_z: 0.095
modes:
  walk: 0
  ef: 1
  body: 2
  joint: 3
